src/netstandard/pkg/NETStandard.Library.pkgproj

```diff
@@ -32,6 +32,10 @@
     <File Include="$(PlaceHolderFile)">
       <TargetPath>lib/netstandard1.0</TargetPath>
     </File>
+
+    <File Include="$(PackagesDir)$(NetStandardLibraryPackage)/$(NetStandardLibraryPackageVersion)/build/netstandard2.0/**">
```

Comment:
You may want to do something to ensure that this path exists and you don't just get an empty wildcard.